stories/typography.stories.tsx

```diff
@@ -0,0 +1,45 @@
+import { Typography } from "@material-ui/core";
+import { Meta, Story } from "@storybook/react";
+import React from "react";
+
+import { Decorator, GuideDecorator } from "../src/utils/Decorator";
+import useGuideStyles from "../src/utils/guideStyles";
+import { Cell } from "./utils/Cell";
+
+const DefaultStory: React.FC = () => {
```

Comment:
I created this story mostly because it's difficult to figure out which MUI variant is used in the design :P